perf/cdcsdk-base.yaml

```diff
@@ -0,0 +1,35 @@
+services:
+  cdcsdk:
+    image: quay.io/yugabyte/cdcsdk-server:0.7.0-SNAPSHOT
```

Comment:
Just wondering, this will always fetch the image tagged with `0.7.0-SNAPSHOT`, would it make sense to change it to `latest` so that if somebody even builds the image using maven plugin, he gets the correct version?